WordPressAuthenticator.podspec

```diff
@@ -1,20 +1,23 @@
 Pod::Spec.new do |s|
   s.name          = "WordPressAuthenticator"
   s.version       = "1.36.0-beta.2"
-  s.summary       = "WordPressAuthenticator implements an easy and elegant way to authenticate your WordPress Apps."
 
+  s.summary       = "WordPressAuthenticator implements an easy and elegant way to authenticate your WordPress Apps."
   s.description   = <<-DESC
                     This framework encapsulates everything required to display the Authentication UI
                     and perform authentication against WordPress.com and WordPress.org sites.
 
-                    Plus: WordPress.com *signup*  is supported.
+                    Plus: WordPress.com *signup* is supported.
                     DESC
 
-  s.homepage      = "https://github.com/wordpress-mobile/WordPressAuthenticator-iOS"
-  s.license       = "GPLv2"
-  s.author        = { "WordPress" => "mobile@automattic.com" }
+  s.homepage      = "http://apps.wordpress.com"
+  s.license       = { :type => "GPLv2", :file => "LICENSE" }
+  s.author        = { "Automattic" => "mobile@automattic.com" }
```

Comment:
For consistency with other pods.

* Some were using `Automattic` as the author name, others were using `WordPress`. Since those pods are authored mainly by the Automattic company, and more importantly the email address associated with that author is an a8c address, I figured it made more sense to use `Automattic` rather than `WordPress` here
* Some other pods are listing multiple authors, like `mobile@automattic.com` but also and/or actual contributors (a12s). [[Example in WordPressUI pod](https://raw.githubusercontent.com/wordpress-mobile/WordPressUI-iOS/develop/WordPressUI.podspec)]. I wonder if we will want to keep them listed too, or if we should just simply use a single `mobile@automattic.com` author for all our pods?

Reply:
It's nice to have your name listen among the authors of a library, but at the same time it's a bit of a maintenance burden. It also leaves the question open as what to do when someone leaves or stops being involved with the project, should they remain in the list because of their contributions so far, or be removed to avoid being contacted about something they're not working on anymore?

Overall, I think a catch-all setup like the one you're proposing makes things more straightforward.

Some of our projects are narrow-focused on WordPress, but other (and maybe more in the future) are more generic. As such, putting everything under the Automattic umbrella makes sense to me. 

As for the authors, it's a matter of keeping things consistent and lowering the amount of ad hoc decision making when modifying the spec 😄

Reply:
Yeah I agree. I really like crediting people who contributed; but that can be done in a separate place — and GitHub already has a Insights->Contributors pane that is updated automatically. So imho keeping only a8c as a single generic author feels simpler especially if we want to standardize stuff.